frameworklauncher/src/main/java/com/microsoft/frameworklauncher/service/Service.java

```diff
@@ -594,34 +601,36 @@ private void launchApplication(FrameworkStatus frameworkStatus, ApplicationSubmi
     statusManager.transitionFrameworkState(frameworkName, FrameworkState.APPLICATION_LAUNCHED);
   }
 
-  private void createApplication(FrameworkStatus frameworkStatus) throws Exception {
+  private void createApplication(FrameworkStatus frameworkStatus, boolean usedToStop) throws Exception {
```

Comment:
>usedToStop [](start = 74, length = 10)

"stopping" would be a more appropriate name I think.

Reply:
but it is better to use verb than noun after to.
And eventually we stop it :)

---
In reply to: [172148127](https://github.com/Microsoft/pai/pull/346#discussion_r172148127) [](ancestors = 172148127)

Reply:
"isPlaceholderApplication" would be better.

---
In reply to: [172154514](https://github.com/Microsoft/pai/pull/346#discussion_r172154514) [](ancestors = 172154514,172148127)